### PR TITLE
Added proper back-press behaviour for cross-thread links

### PR DESCRIPTION
**Pull request by Hual**
Now goes back to the thread where the link was opened.

### CODE REVIEW

**Comment by beschoenen** [comment]
Will this also exit Clover when I opened a thread from say Chrome?

**Comment by Hual** [comment]
No, it only follows in-app links.

**Comment by Floens** [comment]
You forgot to commit all files, the CI check failed.

**Comment by Hual** [comment]
I'm an idiot. Added the missing file and CI build is successful now.

**Comment by Floens** [comment]
This needs to be clean up / looked at first. There are some formatting errors and I feel we need a bit of refactoring first.
This PR is related to #218

File: Clover/app/src/main/java/org/floens/chan/ui/controller/ViewThreadController.java [hunk 1]
```diff
@@ -258,4 +271,21 @@ private void setPinIconState() {
...
+
+    @Override
+    public boolean threadBackPressed() {
+        if(threadFollowerPool.empty()) {
+            return false;
+        } else {
+            Loadable threadLoadable = threadFollowerPool.get();
+            loadThread(threadLoadable, false);
+
+            return true;
```
Comment (andyklimczak) [inline Clover/app/src/main/java/org/floens/chan/ui/controller/ViewThreadController.java]: can be simplified slightly:
```java
    @Override
    public boolean threadBackPressed() {
        if(threadFollowerPool.empty()) {
            return false;
        }
        Loadable threadLoadable = threadFollowerPool.get();
        loadThread(threadLoadable, false);
        return true;
    }
```

**Pull request closed without merging**